pkg/virt-handler/mount-manager/utils/recorder.go

```diff
@@ -0,0 +1,218 @@
+package recorder
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"k8s.io/apimachinery/pkg/types"
+	v1 "kubevirt.io/api/core/v1"
+
+	diskutils "kubevirt.io/kubevirt/pkg/ephemeral-disk-utils"
+	"kubevirt.io/kubevirt/pkg/safepath"
+	"kubevirt.io/kubevirt/pkg/unsafepath"
+)
+
+//go:generate mockgen -source $GOFILE -package=$GOPACKAGE -destination=generated_mock_$GOFILE
+type MountTargetEntry struct {
+	TargetFile string `json:"targetFile"`
+	SocketFile string `json:"socketFile,omitempty"`
```

Comment:
Now do we need to keep checking that the socket file is there for container disks?

Reply:
Sorry, can you elaborate a bit the comment? I don't see any part of [the old code](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-handler/container-disk/mount.go)  where we check for the existence of the socket file.